test_darwin_live.py

```
@@ -0,0 +1,275 @@
+    import zeep
+    print(f"   ✅ ZEEP installed: version {zeep.__version__}")
+except ImportError as e:
+    print(f"   ❌ ZEEP not installed: {e}")
+    print()
+    print("   To install ZEEP:")
+    print("   pip3 install zeep lxml requests")
+    sys.exit(1)
+
+try:
+    import lxml
+    print(f"   ✅ lxml installed")
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)